### PR TITLE
Bug 1463260 - Allow writeDocs to work again

### PR DESCRIPTION
**Pull request by imbstack**
Tests will fail until we land taskcluster/taskcluster-lib-validate#39 and taskcluster/taskcluster-lib-api#100

### CODE REVIEW

File: test/helper.js [hunk 1]
```diff
@@ -94,9 +95,9 @@ exports.withServer = (mock, skipping) => {
     exports.load.cfg('taskcluster.accessToken', null);
     fakeauth.start(testClients, {rootUrl});
 
-    const api = await exports.load('api');
+    await exports.load('api');
```
Comment (djmitche) [inline test/helper.js]: I *think* this can be omitted in this case

Comment (imbstack) [reply test/helper.js]: I thought so too but tests started failing when I removed it, iirc. /me shrugs

Comment (djmitche) [reply test/helper.js]: OK, that's not wholly surprising.. ordering is tricky in tests.

**Comment by djmitche** [comment]
From a writeDocs run:

```json
{
  "version": 0,
  "$schema": "http://schemas.taskcluster.net/base/v1/api-reference.json#",
  "title": "TaskCluster Secrets API Documentation",
  "description": "The secrets service provides a simple key/value store for small bits of secret\ndata.  Access is limited by scopes, so values can be considered secret from\nthose who do not have the relevant scopes.\n\nSecrets also have an expiration date, and once a secret has expired it can no\nlonger be read.  This is useful for short-term secrets such as a temporary\nservice credential or a one-time signing key.",
  "baseUrl": "https://secrets.taskcluster.net/v1/",
  "serviceName": "secrets",
  "entries": [
    {
      "type": "function",
      "method": "put",
      "route": "/secret/<name>",
      "query": [],
      "args": [
        "name"
      ],
      "name": "set",
      "stability": "stable",
      "title": "Set Secret",
      "input": "v1/secret.json#",
      "description": "Set the secret associated with some key.  If the secret already exists, it is\nupdated instead.",
      "scopes": "secrets:set:<name>"
    },
```

So that looks good (I'm not worried about $schema right now).  And one of the schema files:

```json
{
  "$schema": "http://json-schema.org/draft-06/schema#",
  "title": "Secret",
  "description": "Message containing a TaskCluster Secret\n",
  "type": "object",
  "properties": {
    "secret": {
      "description": "The secret value to be encrypted.\n",
      "type": "object"
    },
    "expires": {
      "description": "An expiration date for this secret.\n",
      "type": "string",
      "format": "date-time"
    }
  },
  "additionalProperties": false,
  "required": [
    "secret",
    "expires"
  ],
  "$id": "taskcluster:/schemas/secrets/v1/secret.json#"
}
```

This output is identical to that which will be in the published tarball (lib-docs just untars the tarball internally to implement writeDocs).

I'll let you (@imbstack) click the magic merge button on this PR when you're ready.

**Pull request merged**